src/golang/cmd/server/handler/v2/node_operator_content_get.go

```diff
@@ -0,0 +1,108 @@
+package v2
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"net/http"
+
+	"github.com/aqueducthq/aqueduct/cmd/server/handler"
+	server_resp "github.com/aqueducthq/aqueduct/cmd/server/response"
+	"github.com/aqueducthq/aqueduct/lib/database"
+	"github.com/aqueducthq/aqueduct/lib/repos"
+	"github.com/aqueducthq/aqueduct/lib/response"
+	"github.com/aqueducthq/aqueduct/lib/storage"
+	"github.com/dropbox/godropbox/errors"
+)
+
+// This file should map directly to
+// src/ui/common/src/handlers/NodeOperatorContentGet.tsx
+//
+// Route: /api/v2/workflow/{workflowID}/dag/{dagID}/node/operator/{nodeID}/content
+// Method: GET
+// Params:
+//	`workflowID`: ID for `workflow` object
+//  `dagID`: ID for `workflow_dag` object
+//	`nodeID`: ID for operator object
+// Request:
+//	Headers:
+//		`api-key`: user's API Key
+// Response:
+//	Body:
+//		`response.Content`
+
+type NodeOperatorContentGetHandler struct {
+	nodeGetHandler
+	handler.GetHandler
+
+	Database database.Database
+
+	WorkflowRepo repos.Workflow
+	DAGRepo      repos.DAG
+	OperatorRepo repos.Operator
+}
+
+func (*NodeOperatorContentGetHandler) Name() string {
+	return "NodeOperatorContentGet"
+}
+
+func (h *NodeOperatorContentGetHandler) Prepare(r *http.Request) (interface{}, int, error) {
+	return h.nodeGetHandler.Prepare(r)
+}
+
+func (h *NodeOperatorContentGetHandler) Perform(ctx context.Context, interfaceArgs interface{}) (interface{}, int, error) {
```

Comment:
review will be helpful. This is similar to current `export function` route but some implementation is simpler.